.github/workflows/winget-submission.yml

```diff
@@ -0,0 +1,23 @@
+name: Submit package to Windows Package Manager Community Repository 
+
+on:
+  release:
+    types: [published]
+
+env:
+  version: "$(git describe --tags --abbrev=0)"
+  packageFileName: coder_${version}_windows_amd64.zip
+  packageId: Coder.Coder  # Most probably the id of coder package
```

Comment:
These env vars should be in `UPPER_SNAKE_CASE` format like normal env vars, and should be referenced by `"$NAME"` in the script below instead of using `${env:NAME}` as they have different semantics